valtech/aem-easy-content-upgrade
# Feature/aemaacs compatibility

Testing:

AEM 6.5:
* manual run of scripts - OK
* install hooks - OK
* history search - OK
* service user permissions - OK
* health check - OK
* maintenance task - OK
* health check GUI - OK

AEM cloud local
* manual run of scripts - OK
* install hooks - OK
* history search - DISABLED
* service user permissions - OK
* health check - not existing on cloud
* maintenance task - not existing on cloud
* health check GUI - not existing on cloud

AEM cloud
* manual run of scripts - OPEN
* install hooks - OPEN
* history search - OPEN
* service user permissions - OPEN
* health check - not existing on cloud
* maintenance task - not existing on cloud
* health check GUI - not existing on cloud
